event_survey/README.rst

```diff
@@ -0,0 +1,47 @@
+================
```

Comment:
RST broken.
If you move to readme folder structure https://github.com/OCA/maintainer-tools/tree/master/template/module/readme you can forget about this issues, at least for titles ;)